src/libs/events.js

```diff
@@ -53,6 +53,7 @@ const eventsList = {
   notificationToggle: 'notification:toggle',
   pageView: 'page:view',
   removeBillingAccount: 'billing:project:account:remove',
+  resourceLeave: 'resource:leave',
```

Comment:
Did you remember to update the MixPanel lexicon?

Reply:
I will go and do that- thanks!